app/assets/javascripts/miq_application.js

```diff
@@ -1629,40 +1628,28 @@ function chartData(type, data, data2) {
       var max = _.max(_.flatten(_.tail(data.data.columns).map(_.tail)));
       data.axis.y.tick.values = [0, max];
     }
+
+    if(data.miq.reporting_chart) {
+      data.tooltip.format.name =  function (name, ratio, id, index) { return data.miq.name_table[id]; }
+      data.tooltip.format.title =  function (x) { console.log(data.miq.category_table[x]); return data.miq.category_table[x]; }
```

Comment:
generally, look for the style a bit. is there a reason to make these functions single-liners?